sklearn/feature_extraction/text.py

```diff
@@ -746,6 +784,27 @@ class CountVectorizer(BaseEstimator, VectorizerMixin):
 
         This is only available if no vocabulary was given.
 
+    >>> from sklearn.feature_extraction.text import TfidfVectorizer
```

Comment:
We should illustrate only the CountVectorizer here

Reply:
Fixed with [this commit](https://github.com/scikit-learn/scikit-learn/pull/11534/commits/f7fd421f3454aaee9fe4d8f88e8258e36b8b7ca3).